app/Http/Controllers/Api/v1/Mensa/Controllers/SignupController.php

```diff
@@ -75,7 +77,12 @@ public function signup(Request $request, string $mensaId): ?JsonResponse
 
         $user = $this->currentUser();
         $isAdmin = $user != null && $user->mensa_admin;
-
+        
+        $log = new Log;
```

Comment:
Although `new Log` without parentheses are allowed, I think we should still always do it for consistency